package.json

```diff
@@ -1,6 +1,16 @@
 {
```

Comment:
maybe add `husky` for an automated setup of a precommit hook

Reply:
done